releases/release-1.32/release-notes/maps/pr-127012-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 127012
+releasenote:
+  text: Sent bookmark right now after sending all items in watchCache store.
```

Maybe reword based on the longer description? https://github.com/kubernetes/kubernetes/pull/127012 

```suggestion
  text: Bookmark events are now sent immediately after all items in the watchCache store have been processed, improving consistency in client behavior.
```